docs/DeveloperGuide.md

```diff
@@ -740,58 +739,109 @@ testers are expected to do more *exploratory* testing.
 ### Launch and shutdown
 
 1. Initial launch
-
    1. Download the jar file and copy into an empty folder
+   2. Open up a terminal, and run the following command: `java -jar DocTrack.jar`
+      - Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
-
-1. Saving window preferences
-
+2. Saving window preferences
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
-
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
-
 ### Adding a person
 
+1. Adding a person
+   1. Prerequisites: None
+   2. Test case:`add person n/Elmo p/98765432 e/elmo@sesa.me a/Sesame Street st/recovering t/insurance`
+   3. Expected: A new contact is added to the list.
+   - The status bar shows the following: "New person added: Elmo; Phone: 98765432; 
+   Email: elmo@sesa.me; Address: Sesame Street; Status: recovering; Tags: [insurance]"
+   - The list of persons should now have the new contact inside it.
+    
+
 ### Deleting a person
 
 1. Deleting a person while all persons are being shown
-
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
-
-   1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
-
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
-
-1. _{ more test cases …​ }_
+   1. Prerequisites: Multiple persons in the DocTrack. If they are not shown, do `list person`.
+   2. Test case: `delete person 1`<br>
+      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. 
+   Timestamp in the status bar is updated.
+   3. Test case: `delete person -1`<br>
+         Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+         Expected: Similar to previous.
```

Comment:
Should we specify the expected result?

Reply:
Hm, good question. I feel it would be redundant.

Do you feel this is unclear? If it feels like so to you, I think it's a change we can make!

Reply:
I think can specify it!